src/legacy/core_plugins/kibana/public/visualize/legacy_imports.ts

```diff
@@ -41,33 +41,31 @@ export { PersistedState } from 'ui/persisted_state';
 
 export { npSetup, npStart } from 'ui/new_platform';
 export { IPrivate } from 'ui/private';
```

Comment:
The code for private is moved, lets also move the type and reference it via re-export.

Reply:
thx! will clean up!

Reply:
done